hbook/ntup/ntup_test/ntup_example_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2015 The go-hep Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ntup_test
+
+import (
+	"fmt"
```

Comment:
we try to distinguish b/w imports from the standard library, and the 3rd party ones.
could you split the imports into 2 lists?

something like:
```go
import (
	"fmt"

	"go-hep.org/x/hep/hbook"
	"golang.org/x/exp/rand"
	"gonum.org/v1/gonum/stat/distuv"
)
```

also: no need to actually import `goimports`.
it's a command that you install once and then run over the code source:
```
$> go get golang.org/x/tools/cmd/goimports
```
and then:
```
$> cd go-hep.org/x/hep
$> goimports -w .
```
(but as I said earlier, it's best to hook it up with your favorite editor so it's run behind the scenes whenever you save the file or modify it.)

Reply:
@sbinet, sure. will try to hook it in the editor.